x-pack/plugins/alerting/README.md

```diff
@@ -0,0 +1,297 @@
+# Kibana alerting
+
+The Kibana alerting plugin provides a common place to set up alerts. You can:
+
+- Register types of alerts
+- List the types of registered alerts
+- Perform CRUD actions on alerts
+
+## Terminology
+
+**Alert Type**: A function that takes parameters and fires actions to alert instances.
+
+**Alert**: A configuration that defines a schedule, an alert type w/ parameters, state information and actions.
+
+**Alert Instance**: The instance(s) created from an alert type execution.
+
```

Comment:
A short explanation that builds a mental model might be helpful here, since alerts have quite a few moving pieces. 

I'll take a stab at it: 
```
A Kibana alert detects a condition and fires one or more actions when that condition occurs.  Alerts work by going through the followings steps:
1) Run a periodic check to detect a condition (the check is provided by an Alert Type) 
2) Convert that condition into one or more stateful Alert Instances
3) Map Alert Instances to pre-defined Actions, using templating
4) Execute the Actions
```